test_huggingface_live.py

```
@@ -0,0 +1,313 @@
+    config = HuggingFaceInferenceConfig(
+        model_name="HuggingFaceH4/zephyr-7b-beta",
+        api_key=os.getenv("HF_TOKEN"),
+        max_new_tokens=30,
+        temperature=0.7,
+    )
+
+    builder = MagicMock(spec=Builder)
+
+    try:
+        async with huggingface_inference_langchain(config, builder) as client:
+            print(f"✓ Client created for streaming test")
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)